compiler/rustc_codegen_llvm/src/intrinsic.rs

```diff
@@ -1492,6 +1492,100 @@ fn generic_simd_intrinsic<'ll, 'tcx>(
         return Ok(v);
     }
 
+    if name == sym::simd_load {
```

Comment:
this should be named `simd_masked_load`, since I expect portable-simd to want a unmasked load at some point